docs/code/deployments-kubernetes/example.rego

```diff
@@ -1,15 +1,11 @@
 package example
 
-default deny = false
-
 # Reject objects without a customer label.
-deny {
-    not input.metadata.labels.customer
+deny["Missing customer label"] {
+    not input.tags.customer
 }
 
 # Reject pods referring to images outside the corporate registry.
-deny {
-    input.kind == "Pod"
-    container := input.spec.containers[_]
-    not re_match("^registry.acmecorp.com/.+$", container.image)
+deny["Invalid image registry"] {
```

Comment:
If the goal is to show just OPA and not relate it to the admission controller should we change the policy to not look like the admission requests at all? The changes made are substantial if you know what the admission requests look like, but the policy is still showing the same sort of behavior.

Maybe we just show the same example as the docker one earlier in that page?

Reply:
The thought crossed my mind at the time. I've updated it to avoid lingering confusion. LMKWYT.